Js/Pop.js

```diff
@@ -0,0 +1,118 @@
+    imageLink: './images/Snapshoot-Portfolio(4).jpg',
+    tags: ['HTML', 'CSS', 'Javascript'],
+    role: 'Back End Dev',
+    company: 'CANOPY',
+    year: '2015',
+    sourceLink: '',
+    demoLink: '',
+  },
+  {
+    title: 'Tonic',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    imageLink: './images/Snapshoot-Portfolio(5).jpg',
+    tags: ['HTML', 'CSS', 'Javascript'],
+    role: 'Back End Dev',
+    company: 'CANOPY',
+    year: '2015',
+    sourceLink: '',
+    demoLink: '',
+  },
+  {
+    title: 'Mulit-Post Stories',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    imageLink: './images/Snapshoot-Portfolio(6).jpg',
+    tags: ['HTML', 'CSS', 'Javascript'],
+    role: 'Back End Dev',
+    company: 'CANOPY',
+    year: '2015',
+    sourceLink: '',
+    demoLink: '',
+  },
+];
+
+(() => {
+  cardCover.innerHTML = '';
+  for (let i = 0; i < Arrayprojects.length; i += 1) {
+    let worklist = '';
+    for (let j = 0; j < Arrayprojects[i].tags.length; j += 1) {
+      worklist = `${worklist}<li class="tag">${Arrayprojects[i].tags[j]}</li>`;
+    }
+    cardCover.innerHTML = `${cardCover.innerHTML}
+    <section class="GridCard3">
+    <section class="GridWrapper3">
+        <section class="second-section3">
+            <div>
+                <img src="${Arrayprojects[i].imageLink}" alt="section-eader" class="img1">
+            </div>
+            <div class="sub-section3">
+                <h2 class="mobile-ton">${Arrayprojects[i].title}</h2>               
+                    <ul class="to-do3">                       
+                        <li class="canopy">${Arrayprojects[i].company}</li>
+                        
+                        <li class="back-end">${Arrayprojects[i].role}</li>
+                        
+                        <li class="year">${Arrayprojects[i].year}</li>
+                    </ul>
+                <p class="to-do-para3">${Arrayprojects[i].description}
+                </p>
+                <div class="works">
+                    <ul>
+                     ${worklist}
+                    </ul>
+                </div>
+                <div class="click24">
+                    <button class="prj-view-btn">See Projects</button>
+                </div>
+            </div>
+        </section>
+    </section>
+</section>`;
+  }
```

Comment:
Your popups are breaking and spanning over a lot more screens than I can capture in the screenshots 😂. I could not even fit the image of the popup on the whole screen screenshot. Kindly fix it.
Here is the reference.
![image](https://user-images.githubusercontent.com/33268927/175067508-63c67bef-7137-4390-b196-2aae713f8b7f.png)